logstreams/src/main/java/io/zeebe/logstreams/impl/log/LogStorageAppender.java

```diff
@@ -230,7 +230,7 @@ public void onWriteError(final Throwable error) {
         // Not a failure. It is probably during transition to follower.
         return;
       }
-      onFailure(error);
+      actor.run(() -> onFailure(error));
```

Comment:
speaks anything against moving the actor run into the onFailure?

Reply:
onFailure is also called in handleFailure(). If we use actor.run() there it won't be executed until other submitted tasks are executed. So we are not immediately failing when an unhandled exception happens.

Reply:
ok then leave it like it is thanks